packages/cyberstorm/src/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { Fragment } from "react";
+import styles from "./componentStyles/Breadcrumb.module.css";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faHouse, faSlashForward } from "@fortawesome/pro-regular-svg-icons";
+import { LinkingProvider } from "@thunderstore/components";
+import { LinkLibrary } from "./LinkLibrary";
+
+interface Crumb {
+  label: string;
+  LinkComponent?: (props: any) => React.ReactElement | null; // eslint-disable-line @typescript-eslint/no-explicit-any
+  LinkProps?: Record<string, any>; // eslint-disable-line @typescript-eslint/no-explicit-any
+}
+
+interface Props {
+  parts: Crumb[];
+}
+
+/**
+ * Navigational bread crumbs defining where the user is at the site.
+ */
+export const BreadCrumb: React.FC<Props> = (props) => {
+  if (!props.parts) {
+    return (
+      <div className={`${styles.breadcrumb}`}>
```

Comment:
There's no need for template string here since it's just one class. I think `className={styles.breadcrumb}` should suffice. This is repeated multiple times below.